go/teams/loader.go

```diff
@@ -140,11 +140,23 @@ func (l *TeamLoader) checkArg(ctx context.Context, lArg keybase1.LoadTeamArg) er
 
 // Resolve a team name to a team ID.
 // Will always hit the server for subteams. The server can lie in this return value.
-func (l *TeamLoader) resolveNameToIDUntrusted(ctx context.Context, teamName string) (keybase1.TeamID, error) {
-	// TODO: Resolve the name to team ID.
+func (l *TeamLoader) resolveNameToIDUntrusted(ctx context.Context, teamName keybase1.TeamName) (id keybase1.TeamID, err error) {
 	// For root team names, just hash.
+	if teamName.IsRootTeam() {
+		return teamName.ToTeamID(), nil
+	}
+
 	// For subteams, ask the server.
-	panic("TODO: resolve team name to id")
+	rres := l.G().Resolver.ResolveFullExpression(ctx, fmt.Sprintf("team:%s", teamName.String()))
```

Comment:
On second thought, it looks like this might cache. I don't want to hit a cache. Loading a team by name is an unhappy path anyway and I don't want to worry about races with subteam renames.